app/activity/[id]/components/activity-card.tsx

```diff
@@ -0,0 +1,82 @@
+'use client'
```

Comment:
Looks like this file is dupicated in components and component directory
<img width="317" alt="Screenshot 2024-01-04 at 7 09 41 AM" src="https://github.com/tabi-memo/frontend/assets/66394413/4f5b5313-c321-43c8-ac6a-a9ee26a42221">